### DISCUSSION TITLE
[Projects Beta] Please support proper Note cards with locking & markdown

### DISCUSSION THREAD

**Discussion by erinmez** [community/community#6604] [Projects and Issues]
Hi there,

In order to communicate instructions to board users I would like notes to :
- support Markdown
- be locked so that they are not accidentally removed
- be excluded from the count in the top column counter
- Nice to have: 
be pinned to the top; In a given column I should be able to select 0 or more cards and pin them to the top

Best, Tanju
<img width="363" alt="notecard" src="https://user-images.githubusercontent.com/16493423/140892025-16634f9f-bbb7-48cc-aca9-dc25d846d87b.png">

**Comment by Relequestual** [comment]
> be excluded from the count in the top column counter

Dissagree. I would want the count.
What do you feel that count represents?

**Reply by erinmez** [reply]
In my scenario, I don't want the notes to be counted, because they are not work items (the issues are). Besides, the number of note cards could vary in different columns. With lots of steps (columns), it is easier for me to just quickly glance over the counters and compare them with the WIP limits that are right next to them.

**Reply by tsalright** [reply]
I think there should be three types of cards in the project board. 

- Issues associated with repositories 
- Draft Issues that do not have a repository and are simply Title and Details
- Notes which are simply title with markdown support adding context to columns

Based on your context above, you want notes to provide additional information to the team in the column or use it as a divider for organizing your cards, and that make complete sense. Those notes should never be converted to an issue. Those make sense to not be counted against WIP.

But I would use drafts as placeholders for issues that do not have a repository because it hasn't been created yet. How else do you plan and capture details in a sprint planning/refinement for future work and have them ready for a team to take on before repositories get created?

**Reply by erinmez** [reply]
Your case makes totally sense. The current draft feature is fine and it should be included in the counter (as it is). 

I guess I should have used a wireframe instead of a draft issue in the image in order to convey what I would like to have for "Note cards".
